pygsti/report/plotly_plot_ex.py

```diff
@@ -103,11 +103,6 @@ def plot_ex(figure_or_data, show_link=True, link_text='Export to plot.ly',
         aspect_ratio = orig_width / orig_height
     else: aspect_ratio = None
 
-    #Remove original dimensions of plot so default of 100% is used below
-    # (and triggers resize-script creation)
-    if orig_width: del fig['layout']['width']
-    if orig_height: del fig['layout']['height']
-
```

Comment:
This change is related to the fix I found for the figure sizing issues I discussed live in the new per-sequence detail plots.